sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/ecert-full-time-process-integration.scheduler.e2e-spec.ts

```diff
@@ -465,5 +468,193 @@ describe(
         }),
       ).toBe(true);
     });
+    it("Should not generate disbursement if the Student assessment contains PD/PPD application flag=yes and Student profile PD/PPD missing approval", async () => {
+      // Arrange
+
+      // Student with valid SIN.
+      const student = await saveFakeStudent(
+        db.dataSource,
+        {},
+        {
+          initialValue: {
```

Comment:
Btw. not requested is default status and doesn't need to be explicitly set. 
![image](https://github.com/bcgov/SIMS/assets/54600590/550043da-ed35-4f73-a1a2-f267326fc70d)

Reply:
Yes you are right, I explicitly mentioned for better understanding of the test case.